ibtk/include/ibtk/AppInitializer.h

```diff
@@ -56,18 +56,22 @@ class AppInitializer : public SAMRAI::tbox::DescribedClass
 
     /*!
      * Return a boolean value indicating whether this is a restarted run.
+     * Defined as true when program recognizes command line arguments for
+     * restart_restore_num and restart_dump_dirname.
      */
     bool isFromRestart() const;
 
     /*!
      * Return the restart directory.  If we are not starting from restart,
-     * this method returns an empty string.
+     * this method returns an empty string. Passed into program in the command
+     * line as argv[2].
```

Comment:
We should clarify here that `argv` is the array of command-line arguments: i.e., its the second argument to the executable.